stdlib/public/core/Arrays.swift.gyb

```diff
@@ -151,7 +151,7 @@ if True:
 /// Here's an implementation of those steps:
 ///
 ///     if let i = absences.index(where: { $0 > 0 }) {                      // 1
-///         let absencesAfterFirst = absences.suffix(from: i + 1)           // 2
+///         let absencesAfterFirst = absences[(i + 1)...]                   // 2
```

Comment:
Oh, rats.  Yes, they are necessary because only binary operators have precedence, and unary operators bind more tightly.  That rots:

```swift
i+1...j-1 // OK
i+1...     // Error :(
```

Someone should file a Jira.